doc/nrf/device_guides/working_with_nrf/nrf70/developing/power_profiling.rst

```diff
@@ -0,0 +1,81 @@
+.. _ug_nrf70_developing_power_profiling:
+
+Power profiling of nRF7002 DK
+#############################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Power Profiler Kit II (PPK2) of Nordic Semiconductor can be used in conjunction with the nRF7002 DK to measure the power consumption of nRF70 Series devices in Low-power mode.
```

Comment:
Strictly speaking, the PPK can be used to measure the power consumption of nRF70 Series in ANY mode :)

Of course, what is interesting is the power consumption in low power mode.

Let's just rephrase this sentence a bit, to get rid of this issue.